src/main/java/training/TableCreator.java

```diff
@@ -1,49 +1,69 @@
-package training;
-
-import com.amazonaws.auth.AWSStaticCredentialsProvider;
-import com.amazonaws.auth.BasicAWSCredentials;
-import com.amazonaws.regions.Regions;
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
-import com.amazonaws.services.dynamodbv2.document.DynamoDB;
-import com.amazonaws.services.dynamodbv2.document.Table;
-import com.amazonaws.services.dynamodbv2.model.*;
-
-import java.util.Arrays;
-
-public class TableCreator {

```

Comment:
Sorry I did not ask you to create table. Manualy creat table and just write the code to insert items

Reply:
but the method create() is intserting items

Reply:
First of all why it is in tablecreator class , you need to create separate classes

Reply:
First of all why it is in tablecreator class , you need to create separate classes

Reply:
by manually create table u mean to say directly from aws website?